standards/python.md

```diff
@@ -0,0 +1,50 @@
+# Python Guide
+
+1. [Linting](#linting)
+1. [Imports](#imports)
+3. [Python versions](#python-versions)
+
+## Linting
+
+All of our Python code should match the [PEP8 style guide](https://www.python.org/dev/peps/pep-0008/) and the [Django coding style guidelines](https://docs.djangoproject.com/en/dev/internals/contributing/writing-code/coding-style/). 
+
+We follow [PEP8's line length](https://www.python.org/dev/peps/pep-0008/#maximum-line-length) instead of Django's for consistency with our other development standards. We follow [Django's recommended import ordering in Django projects](https://docs.djangoproject.com/en/dev/internals/contributing/writing-code/coding-style/#imports).
+
+For linting we recommend [flake8](http://flake8.pycqa.org/en/latest/), and provide a [sample flake8 configuration](../.flake8).
+Exceptions to PEP8 are found in our , and include:
+
+- `E731`, we allow assignment of `lambda` expressions
+- `W503`, we allow line breaks after binary operators
+
+On the whole, generated python files like Django migrations can be safely ignored. 
+
+Both the flake8 and isort configurations can go into a `tox.ini` file under `[flake8]` and `[isort]` headers respectively.
+
+It is highly recommended to configure flake8 linting in your editor of choice to use the configuration file above or one provided in the project you're working on.
+
+## Imports
+
+See [Imports in PEP8](https://www.python.org/dev/peps/pep-0008/#imports) and [Imports in Django Coding Style](https://docs.djangoproject.com/en/dev/internals/contributing/writing-code/coding-style/#imports).
+
+We use [isort](https://github.com/timothycrosley/isort) to lint imports to comply with both style guidelines, and provide a [sample isort configuration](../.isort.cfg). This configuration attempts to enforce the following conventions:
+
+- Multiple-line `from` imports become grouped within parenthesis with one imported name per-line and a trailing comma for the last imported name.
+- Generated Python files like Django migrations can be safely ignored. 
+- Consider `six` to be equivelent to `__future__` imports.
+- Django imports are included as their own section between the standard library and third-party imports.
+- For Wagtail-specific projects like [cfgov-refresh](https://github.com/cfpb/cfgov-refresh), Wagtail imports are included as their own section between Django and third-party imports.
+
+## Python versions
+
+As we transition from Python 2.7 to Python 3 it is highly recommended to test against both Python 2.7 and Python 3.6. 
+
+We recommend using [tox](https://tox.readthedocs.io/en/latest/) for matrix testing, and provide a [sample tox configuration](../tox.ini) that tests against Python 2.7 and 3.6 (and Django versions 1.8-1.11).
```

Comment:
Can we recommend testing against latest and LTS versions of Django instead of all of these? Testing against 1.9, 1.10 might not add much. Additionally, since Django 2.0 should be released Real Soon Now, is it worth it to add that here as well?

Reply:
2.0 will probably be released right after we get this merged, but I think we should wait until it is.